include/tbb/parallel_execution_tbb.h

```diff
@@ -30,73 +30,124 @@
 
 namespace grppi{
 
-/** @brief Set the execution mode to parallel with threading building blocks
- *    (TBB) framework implementation
+/** 
+ \brief TBB parallel execution policy.
```

Comment:
Shouldn't you say Intel TBB? This is just a minor issue, but the sooner it is solved the better.